wordpress-mobile/WordPress-Android
# Gutenberg informative Dialog

This PR fixes the first part of #9034 by showing an informative dialog when the user edits a GB post. This informative dialog is shown on the screen only once.


@iamthomasbishop Thomas would you mind to check the copy?
![screenshot_1550075256](https://user-images.githubusercontent.com/518232/52728615-e50f0c00-2fb7-11e9-97b0-cab3c3e115ac.png)



**Test 1**
- Start the app
- Edit a post that was made on the web (or in the mobile apps) by using the old editor
- Aztec should start (verify that it's Aztec that comes up - look for the "H" toolbar button which only appears in Aztec's case)

**Test 2**
- Start the app
- Edit a post that was made with Gutenberg
- Mobile GB should appear and the info dialog shown

**Test 3**
- Repeat Test 2, but this time the info dialog should not appear on the screen


Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
